images/bootstrap-legacy/Dockerfile

```diff
@@ -0,0 +1,118 @@
+# Copyright 2017 The Kubernetes Authors.
+# Copyright 2021 The KubeVirt Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Includes basic workspace setup, with gcloud and a bootstrap runner
+FROM fedora:36
```

Comment:
maybe use a specific hash / mirror or use specific version to quay.io (it will also be better to use quay one) ?
so it will be more deterministic,
else between versions also the fedora is changed and it is harder to follow
(especially now that we are going to use it less often it might be better to pin it)

Since we run like this for long time, it can be done on follow PR in case desired

Reply:
This can be handled in a follow on - it shouldn't be using the short-name for the image anyway.